lib/openid_connect/token_endpoint.rb

```diff
@@ -0,0 +1,29 @@
+class TokenEndpoint
+  attr_accessor :app
+  delegate :call, to: :app
+
+  def initialize
+    @app = Rack::OAuth2::Server::Token.new do |req, res|
+      client = Client.find_by_identifier(req.client_id) || req.invalid_client!
+      client.secret == req.client_secret || req.invalid_client!
+      case req.grant_type
+        when :client_credentials
+          res.access_token = client.access_tokens.create!.to_bearer_token
+        when :authorization_code
+          authorization = client.authorizations.valid.find_by_code(req.code)
+          req.invalid_grant! if authorization.blank? || !authorization.valid_redirect_uri?(req.redirect_uri)
+          access_token = authorization.access_token
+          res.access_token = access_token.to_bearer_token
+          if access_token.accessible?(Scope::OPENID)
+            res.id_token = access_token.account.id_tokens.create!(
+                client: access_token.client,
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Indent the first parameter one step more than the previous line.